hack/ci-helper.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET_BRANCH=${TARGET_BRANCH:-main}
+
+function rebase_atop_of_the_latest_target_branch() {
+	if [ -n "${TARGET_BRANCH}" ]; then
```

Comment:
Would you please check the `GITHUB_WORKSPACE` is set and non-empty to abort easier in case someone runs this outside of git environment?

Reply:
Ah, it's actually there, please ignore this message